spec/requests/users/create_spec.rb

```diff
@@ -79,7 +79,7 @@
     context 'when passwords don\'t match' do
       let(:password)              { 'shouldmatch' }
       let(:password_confirmation) { 'dontmatch' }
-      let(:new_user)              { User.find_by(email: email) }
+      let(:new_user)              { User.find_by(email:) }
```

Comment:
hmm looks like this change was not needed right? Probably this test is broken rn

Reply:
Noticed this pattern in a lot of test. You need to add the variable back (like `email: email`)

Reply:
fixed in last commit